docs/notebooks/getting_started/particles.ipynb

```diff
@@ -0,0 +1,903 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8986cef8",
+   "metadata": {},
+   "source": [
+    "### Antiparticles\n",
+    "\n",
+    "[Particle]: ../../api/plasmapy.particles.particle_class.Particle.rst\n",
+    "\n",
+    "We can get antiparticles of fundamental particles by using the `antiparticle` attribute of a [Particle]."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6663952",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "electron.antiparticle"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf40e592",
+   "metadata": {},
+   "source": [
+    "[Particle]: ../../api/plasmapy.particles.particle_class.Particle.rst\n",
+    "\n",
+    "We can also use the tilde operator on a [Particle] to get its antiparticle."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "180a1edf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "~proton"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8500bc48",
+   "metadata": {},
+   "source": [
+    "### Ionization and recombination\n",
+    "\n",
+    "[Particle]: ../../api/plasmapy.particles.particle_class.Particle.rst\n",
+    "\n",
+    "The `recombine` and `ionize` methods of a [Particle] representing an ion or neutral atom will return a different [Particle] with fewer or more electrons."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "186b1dfa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deuterium = Particle(\"D 0+\")\n",
+    "deuterium.ionize()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ee95241",
+   "metadata": {},
+   "source": [
+    "When provided with a number, these methods tell how many bound electrons to add or remove."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "242b53bd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "alpha = Particle(\"alpha\")\n",
+    "alpha.recombine(2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ebf2176",
+   "metadata": {},
+   "source": [
+    "[Particle]: ../../api/plasmapy.particles.particle_class.Particle.rst\n",
+    "\n",
+    "If the ``inplace`` keyword is set to `True`, then the [Particle] will be replaced with the new particle."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb2716c0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "argon = Particle(\"Ar 0+\")\n",
+    "argon.ionize(inplace=True)\n",
+    "print(argon)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5302bef",
+   "metadata": {},
+   "source": [
+    "## Custom particles"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c8c2a63",
+   "metadata": {},
+   "source": [
+    "[CustomParticle]: ../../api/plasmapy.particles.particle_class.CustomParticle.rst\n",
+    "\n",
+    "Sometimes we want to use a particle with custom properties.  For that we can use [CustomParticle]."
```

Comment:
Maybe note that custom particles are only valid input for some of the Particle functionality (e.g. I assume you can't do nuclear reactions with these?)